enterprise/cmd/cody-gateway/internal/actor/productsubscription/productsubscription.go

```diff
@@ -55,7 +55,7 @@ func NewSource(logger log.Logger, cache httpcache.Cache, dotComClient graphql.Cl
 	}
 }
 
-func (s *Source) Name() string { return codygateway.ProductSubscriptionActorSourceName }
+func (s *Source) Name() string { return string(codygateway.ActorSourceProductSubscription) }
```

Comment:
We could also change the interface of the actor source:

```suggestion
func (s *Source) Name() codygateway.ActorSource { return codygateway.ActorSourceProductSubscription }
```